server/src/test/java/org/elasticsearch/action/admin/cluster/node/tasks/TasksIT.java

```diff
@@ -725,12 +724,6 @@ public void testTasksWaitForAllTask() throws Exception {
     }
 
     public void testTaskStoringSuccesfulResult() throws Exception {
-        // Randomly create an empty index to make sure the type is created automatically
```

Comment:
I removed this because it made writing the assertions about setting the origin very difficult and it isn't really a supported configuration.